src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2018-2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+
+# Some useful test classes inherited from avocado.Test
+
+from __future__ import print_function
+
+import os
+import json
+
+from avocado import Test as avocadoTest
+from avocado import skip
+
+import fault_config_utils
+import AgentUtils
+import server_utils
+import write_host_file
+from daos_api import DaosContext, DaosLog
+
+# pylint: disable=invalid-name
+def skipForTicket(ticket):
```

Comment:
Also minor thing but for PEP8/consistency purposes, I think we want to stick with snake_case for function names, CapWords for classes.